examples/basic/contact-bot.py

```diff
@@ -0,0 +1,69 @@
+"""contact bot"""
+import asyncio
+import logging
+import os
+from typing import Optional
+
+from wechaty_puppet import ScanStatus, ContactType  # type: ignore
```

Comment:
Can we re-export the `ScanStatus` and `ContactType` from the `wechaty` module, so that we can import everything from `wechaty` so that the end-user will never care about the `wechaty_puppet` anymore?

Reply:
Great Idea, We can re-export all of the `**Type`, `**Status` data structure from the `wechaty` module. This will block out the `wechaty_puppet` in some way.